hoot-services/Makefile

```diff
@@ -62,6 +62,7 @@ clean-db-force:
 	rm -rf ./src/main/java/hoot/services/db2
 
 test: config
+	echo Running services tests...
```

Comment:
This will add extra text when running with `-s`, I think Makefiles should be as close to silent as we can make them with -s.